operator/v1/types_ingress.go

```diff
@@ -545,6 +545,22 @@ type AWSLoadBalancerParameters struct {
 	//
 	// +optional
 	NetworkLoadBalancerParameters *AWSNetworkLoadBalancerParameters `json:"networkLoadBalancer,omitempty"`
+
+	// subnets specifies the list of subnets for the load balancer to
+	// route traffic to. The values can be either a subnet ID or name.
+	// In order for the load balancer to be provisioned with subnets:
+	// * Each subnet must exist.
+	// * Each subnet must be from a different availability zone.
+	// * The load balancer service must be deleted.
+	//
+	// When omitted, the subnets will be auto-discovered per availability zone.
```

Comment:
If I understand this correctly:
```suggestion
	// If omitted, the subnets will be auto-discovered per availability zone.
	// When subnets are auto-discovered, they do not propagate to this list.
```